packages/flutter_tools/lib/src/commands/build_ios.dart

```diff
@@ -294,6 +294,11 @@ abstract class _BuildIOSSubCommand extends BuildSubCommand {
       globals.printStatus(
         'A summary of your iOS bundle analysis can be found at: ${outputFile.path}',
       );
+      globals.printStatus(
```

Comment:
sorry, if you add other platforms on this PR, please add corresponding tests for them too
https://cs.opensource.google/search?q=%22--analyze-size%22&ss=flutter

Reply:
done